project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject clojurewerkz/elastisch "2.1.0"
+(defproject clojurewerkz/elastisch "2.1.1-SNAPSHOT"
```

Comment:
Never update versions in a pull request. It's up to the maintain to decide when to do it and how.